src/wrappers.jl

```diff
@@ -425,6 +432,9 @@ end
 function FeatureCollection(fc::FeatureCollection; crs=crs(fc), extent=extent(fc))
     FeatureCollection(parent(fc), crs, extent)
 end
+function FeatureCollection(fc::Feature; crs=crs(fc), extent=extent(fc))
```

Comment:
Actually this isn't quite right. Instead you want to check `isfeature` in the method below